keras/preprocessing/image.py

```diff
@@ -429,6 +444,7 @@ def __init__(self,
                  rotation_range=0.,
                  width_shift_range=0.,
                  height_shift_range=0.,
+                 brightness_range=0.,
```

Comment:
This is not a range

Reply:
@Dref360 I was just following the others. Will this still work? You just have to provide a tuple of floats.

Reply:
I would put None in this case. I know Python is not typed, but if the default is 0 and we expect a tuple, it's confusing

Reply:
Alright